### DISCUSSION TITLE
Which command option enables tighter segment timing for VTT cue output?

### DISCUSSION THREAD

**Discussion by prarsena** [ggml-org/whisper.cpp#1347] [Q&A]
I'm having a hard time understanding some of the command options from the help file. 

I have been experimenting with max length and found a pretty good value at 50. 

`./main -t 6 -ml 50 -sow true -m models/ggml-small.en.bin -f "${script_dir}/${wav_file}"\
 -ovtt true -of "${script_dir}/${transcription_file}"`

But I'm looking to have the VTT Cues be more tightly aligned with when they are spoken. Currently, the timing of the caption starts at 00:00, and the second caption begins immediately after, regardless of any pauses in the dialogue: 

```
WEBVTT

00:00:00.000 --> 00:00:03.010
 *Singing* And I learned to get along and so you're

00:00:03.010 --> 00:00:06.220
 back. From outer space. I just walked in to find

00:00:06.220 --> 00:00:09.040
 you here with that sad look upon your face. I

00:00:09.040 --> 00:00:12.110
 should have changed that stupid lock. I should

00:00:12.110 --> 00:00:15.070
 have made you leave your key. If I had known for
```

Is there a command option to align the value of the cues with the words as they are spoken? This would allow the captions to disappear when dialogue is not being spoken.

**Comment by mrfragger** [comment]
there is no empty time or pauses in whisper
